From 52ef8d946faefe4c5dfcec86d47aaed17d78a59c Mon Sep 17 00:00:00 2001
From: Graeme Gregory <graeme@xora.org.uk>
Date: Tue, 10 Jan 2023 15:27:57 +0000
Subject: [PATCH] ldold : hack to not use xtc68 install inside gcc builds

According to the docs gcc and xtc68 object files are not 100% compatible
so don't mix them in the same build.

Signed-off-by: Graeme Gregory <graeme@xora.org.uk>
---
 src/ld/ldold.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/ld/ldold.c b/src/ld/ldold.c
index 0cc38b3..21c0c31 100644
--- a/src/ld/ldold.c
+++ b/src/ld/ldold.c
@@ -1664,7 +1664,7 @@ void command_line(int *xac, char ***xav, char **paths, char *lib_arr) {
     if (ldd != NULL) {
       paths[num_paths++] = strdup(ldd);
     }
-    paths[num_paths++] = "/usr/local/share/qdos/lib";
+    paths[num_paths++] = "/usr/local/qdos-gcc/lib";
     for (int i = 0; i < num_paths; i++) {
       remove_trailing_slash(paths[i]);
     }
-- 
2.39.0

